packages/apps-config/src/api/spec/uart.ts

```diff
@@ -21,17 +21,20 @@ const definitions: OverrideBundleDefinition = {
         Value: 'Vec<u8>',
         ReasonIndex: 'u32',
         LottoIndex: 'u32',
+        MemberId: 'u64',
+        ProposalId: 'u64',
+        TokenBalance: 'Balance',
         LotteryKind: {
-          _enum: {
-            Routine: 'Null',
-            TreasuryFunded: 'ReasonIndex'
-          }
+          _enum: [
+            'Routine',
+            'TreasuryFunded(ReasonIndex)'
+          ]
         },
         LottoResult: {
-          _enum: {
-            Routine: '(AccountId, Balance)',
-            TreasuryFunded: 'Balance'
-          }
+          _enum: [
+            'Routine(AccountId, Balance)',
+            'TreasuryFunded(Balance)'
```

Comment:
As above, the original types was correct, this makes it invalid (or rather a c-like enum which won't be able to decode the values)

Reply:
ok